demisto_sdk/commands/update_release_notes/update_rn_manager.py

```diff
@@ -62,29 +62,65 @@ def manage_rn_update(self):
+                    file_path = str(file[1])
+
+                else:
+                    file_path = str(file)
+
+                file_pack_name = get_pack_name(file_path)
+                if file_pack_name not in self.given_pack:
+                    continue
+
+                filtered_set.add(file)
+
+        return validate_manager.filter_to_relevant_files(filtered_set)
+
+    def filter_files_from_git(self, modified_files: set, added_files: set, renamed_files: set,
+                              validate_manager: ValidateManager):
+        """
+        Filter the raw file sets to only the relevant files for RN
+        """
+        filtered_modified, old_format_files = self.filter_to_relevant_files(modified_files, validate_manager)
+        filtered_renamed, _ = self.filter_to_relevant_files(renamed_files, validate_manager)
+        filtered_modified = filtered_modified.union(filtered_renamed)
+        filtered_added, new_files_in_old_format = self.filter_to_relevant_files(added_files, validate_manager)
+        old_format_files = old_format_files.union(new_files_in_old_format)
+        return filtered_modified, filtered_added, old_format_files
+
+    def setup_validate_manager(self):
+        return ValidateManager(skip_pack_rn_validation=True, prev_ver=self.prev_ver,
+                               silence_init_prints=True, skip_conf_json=True, check_is_unskipped=False,
+                               file_path=self.given_pack)
+
+    def get_git_changed_files(self) -> Tuple[set, set, set]:
         """ Get the changed files from git (added, modified, old format, metadata).
 
             :return:
-                4 sets:
                 - The filtered modified files (including the renamed files)
                 - The filtered added files
-                - The changed metadata files
                 - The modified old-format files (legacy unified python files)
         """
         try:
-            validate_manager = ValidateManager(skip_pack_rn_validation=True, prev_ver=self.prev_ver,
-                                               silence_init_prints=True, skip_conf_json=True, check_is_unskipped=False)
+            validate_manager = self.setup_validate_manager()
             if not validate_manager.git_util:  # in case git utils can't be initialized.
                 raise git.InvalidGitRepositoryError('unable to connect to git.')
             validate_manager.setup_git_params()
-            if self.given_pack:
-                # The Validator prints errors which are related to all changed files that were changed against prev
-                # version. When the user is giving a specific pack to update, we want to suppress the error messages
-                # which are related to other packs.
-                with suppress_stdout():
-                    return validate_manager.get_changed_files_from_git()
-            return validate_manager.get_changed_files_from_git()
+
+            with suppress_stdout():
```

Comment:
But what about when pack input was not given? We wouldn't want to suppress STDOUT then wouldn't we? Or maybe im not familiar with the logic here

Reply:
good point fixing